src/Collection.ts

```diff
@@ -57,24 +56,17 @@ export class Collection<T extends FieldModel<unknown>> {
     async get(key: string): Promise<FieldType<T> | undefined>;
     async get<P = unknown>(key: string, path: string): Promise<P | undefined>;
     async get<P>(key: string, path?: string) {
-        const { data } =
-            (await this.collection.findOne({
-                ID: key
-            })) || {};
+        const { data } = (await this.collection.findOne({ ID: key })) || {};
 
         if (data) {
             this.model.validate(data);
-
             if (path) {
-                if (typeof data !== "object") {
-                    throw new Error("Received value must be an 'object'");
-                }
-
+                if (typeof data !== "object") throw new Error("Received value must be an 'object'");
                 return dots.get<P>(data, path);
             }
-        }
 
-        return data || undefined;
+            return data;
+        } else return null;
```

Comment:
What's the difference between `if (data) return data; else return null;` and `return data || null;`